kubectl-plugin/waiops-status.md

```diff
@@ -8,33 +8,156 @@ https://kubernetes.io/docs/tasks/extend-kubectl/kubectl-plugins/
+ir-core-operator                                                                  ibm-aiops-ir-core                    ibm-cp-waiops-catalog   3.3-dev
+ir-lifecycle-operator                                                             ibm-aiops-ir-lifecycle               ibm-cp-waiops-catalog   3.3-dev
+redis                                                                             ibm-cloud-databases-redis-operator   ibm-cp-waiops-catalog   v1.4
+vault                                                                             ibm-vault-operator                   ibm-cp-waiops-catalog   3.3-dev
+______________________________________________________________
+Subscriptions from ibm-common-services namespace:
+
+NAME                                              PACKAGE                                           SOURCE                  CHANNEL
+ibm-cert-manager-operator                         ibm-cert-manager-operator                         ibm-cp-waiops-catalog   v3
+ibm-common-service-operator                       ibm-common-service-operator                       ibm-cp-waiops-catalog   v3
+ibm-commonui-operator                             ibm-commonui-operator-app                         ibm-cp-waiops-catalog   v3
+ibm-crossplane-operator-app                       ibm-crossplane-operator-app                       ibm-cp-waiops-catalog   v3
+ibm-crossplane-provider-kubernetes-operator-app   ibm-crossplane-provider-kubernetes-operator-app   ibm-cp-waiops-catalog   v3
+ibm-events-operator                               ibm-events-operator                               ibm-cp-waiops-catalog   v3
+ibm-iam-operator                                  ibm-iam-operator                                  ibm-cp-waiops-catalog   v3
+ibm-ingress-nginx-operator                        ibm-ingress-nginx-operator-app                    ibm-cp-waiops-catalog   v3
+ibm-licensing-operator                            ibm-licensing-operator-app                        ibm-cp-waiops-catalog   v3
+ibm-management-ingress-operator                   ibm-management-ingress-operator-app               ibm-cp-waiops-catalog   v3
+ibm-mongodb-operator                              ibm-mongodb-operator-app                          ibm-cp-waiops-catalog   v3
+ibm-namespace-scope-operator                      ibm-namespace-scope-operator                      ibm-cp-waiops-catalog   v3
+ibm-platform-api-operator                         ibm-platform-api-operator-app                     ibm-cp-waiops-catalog   v3
+ibm-zen-operator                                  ibm-zen-operator                                  ibm-cp-waiops-catalog   v3
+operand-deployment-lifecycle-manager-app          ibm-odlm                                          ibm-cp-waiops-catalog   v3
+______________________________________________________________
+ODLM pod current status:
```

Comment:
I think we want to look at the operandrequests to see if they are in running state also in the ODLM space.